src/components/Header.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Navbar, Nav } from 'react-bootstrap';
+import './header.css';
+
+function Header({logoImg, width, height, heading}) {
+
+    return (
+        <div>
+            <Navbar bg="dark" variant="dark">
+                <Navbar.Brand href="/Home" className="navBrand">
+                    <img
+                        alt=""
+                        src={logoImg}
+                        width={width ? width: 30}
+                        height={height ? height : 30}
+                        className="d-inline-block align-top"
+                    />{' '}
+                    {heading}
```

Comment:
Also, add default Heading prop as LOGO or BRAND-NAME if anyone does not pass any value to heading prop in App.js